Update PTT Default Config

devilflippy opened the pull request:
Add compat for A.E.S
Svetlana and Evelyn =>  Mechanic Stash
Anastasia => Factory Cellar - Reserve Bunker Hermetic Door

devilflippy committed:
Update PTT Default Config

Add compat for A.E.S
Svetlana and Evelyn =>  Mechanic Stash
Anastasia => Factory Cellar - Reserve Bunker Hermetic Door

guillaumearm commented on user/mods/Trap-PathToTarkov/config/config.json:
```diff
@@ -300,16 +307,94 @@
+    "Svetlana": {
+      "// mod integration for Svetlana": true,
+      "// Svetlana is available at MechanicStash": true,
+      "disable_warning": true,
```
```suggestion
      "disable_warning": false,
```

I think this should be false, it's possible than the load order has an impact and with the warning we can be aware of an issue.

I put this to true in a default config when some people may havn't downloaded the custom trader (and so this avoid polluting his logs with unnecessary warning)

guillaumearm commented on user/mods/Trap-PathToTarkov/config/config.json:
```diff
@@ -300,16 +307,94 @@
+    "Evelyn": {
+      "// mod integration for Evelyn": true,
+      "// Evelyn is available at Hideout": true,
+      "disable_warning": true,
```
same here, this can be false

guillaumearm commented on user/mods/Trap-PathToTarkov/config/config.json:
```diff
@@ -300,16 +307,94 @@
+        "ru": "В лодке"
+      },
+      "access_via": [
+        "AnastasiaStash"
```
```suggestion
        "AnastasiaHideout"
```

guillaumearm commented on user/mods/Trap-PathToTarkov/config/config.json:
```diff
@@ -87,6 +87,13 @@
       "access_via": ["ShorelineWoodsStash"]
+    },
+    {
+      "id": "WelcomeToTarkov_Lotus_stash",
+      "size": 22,
+      "access_via": [
+        "LotusStash"
```
```suggestion
      "id": "WelcomeToTarkov_Anastasia_stash",
      "size": 22,
      "access_via": [
        "AnastasiaHideout"
```

guillaumearm commented on user/mods/Trap-PathToTarkov/config/config.json:
```diff
@@ -300,16 +307,94 @@
+    "Anastasia": {
+      "// mod integration for Anastasia": true,
+      "// Anastasia is available at AnastasiaHideout": true,
+      "disable_warning": true,
```
same here

devilflippy committed:
Update PTT Default config.json

Add A.E.S
Svetlana/Evelyn Hideout
Anastasia Factory Cellar - Reserve - Hermetic Bunker door

guillaumearm commented on user/mods/Trap-PathToTarkov/config/config.json:
```diff
@@ -412,6 +498,10 @@
       "woods": ["Bridge Car"]
-    }
+    },
+    "AnastasiaStasht": {
```
```suggestion
    "AnastasiaStash": {
```

guillaumearm reviewed (changes requested):
Derniere passe ;-)

devilflippy committed:
Update user/mods/Trap-PathToTarkov/config/config.json

Co-authored-by: Guillaume ARM <trapcodien@gmail.com>
